raspbian/install-raspbian.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+BOOTDISK=$1
```

Comment:
Check the correct number of parameters.
Check if parameters are valid.
If something is not valid, return with a help text explaining how the script works.